presto-docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -143,6 +143,50 @@ will function as both a coordinator and worker, use this configuration:
     discovery-server.enabled=true
     discovery.uri=http://example.net:8080
 
+For disaggregated coordinator setup, you can use below minimal configuration:
```

Comment:
How many resource managers and coordinators can I have? How does this functionality work? I think there isn't quite enough information here from someone to deploy and operate the feature and know how/why.

Reply:
Added some details to help understand the functionality.

Reply:
Seems the push didn't worked last time, updated now for real.